src/assets/i18n/en.json

```diff
@@ -362,6 +364,8 @@
         "creationDate": "Creation Date",
         "duration": "Duration",
         "errorLoadingEntry": "Failed to load entry data",
+        "score": "Score",
+        "viewDetails": "View Details",
```

Comment:
@diamond-darrell  - should be "Video Details"

Reply:
fixed